### PR TITLE
Issue3

### PR DESCRIPTION
**Pull request by keerthi-mtalkz**

### CODE REVIEW

File: src/pages/organization/addOrganization.js [hunk 1]
```diff
@@ -103,13 +118,21 @@ const addOrganization = () => {
         <span className="text-default">Phone</span>
         <input
           name="phone"
-          type="text"
+          type="tel"
           ref={register({ required: true })}
           className="form-input mt-1 text-xs block w-full bg-white"
           placeholder="Enter organization phone number"
           required
+          pattern="^\d{10}$"
```
Comment (gaurav-mtalkz) [inline src/pages/organization/addOrganization.js:126]: Do not add custom pattern after you've already specified the type.
We allow between 7 and 13 digits for this.
image

File: src/pages/organization/addOrganization.js [hunk 2]
```diff
@@ -95,6 +104,12 @@ const addOrganization = () => {
```
Comment (gaurav-mtalkz) [inline src/pages/organization/addOrganization.js:104]: Email is also optional

File: src/pages/organization/addOrganization.js [hunk 3]
```diff
@@ -127,6 +150,12 @@ const addOrganization = () => {
           required
```
Comment (gaurav-mtalkz) [inline src/pages/organization/addOrganization.js:150]: Address is optional

File: src/pages/organization/addOrganization.js [hunk 4]
```diff
@@ -142,6 +171,12 @@ const addOrganization = () => {
          required
```
Comment (gaurav-mtalkz) [inline src/pages/organization/addOrganization.js:171]: Add a constraint that it should have only letters and numbers, and max length 15

File: src/pages/organization/update/[updateID].js [hunk 5]
```diff
@@ -129,6 +141,12 @@ const updateID = () => {
```
Comment (gaurav-mtalkz) [inline src/pages/organization/update/[updateID].js:141]: Email is optional

File: src/pages/organization/update/[updateID].js [hunk 6]
```diff
@@ -137,14 +155,22 @@ const updateID = () => {
...
                 <input
                   name="phone"
-                  type="text"
+                  type="tel"
                   ref={register({ required: true })}
                   className="form-input mt-1 text-xs block w-full bg-white"
                   placeholder="Enter your phone no"
                   defaultValue={res.phone}
                   required
+                  pattern="^\d{10}$"
```
Comment (gaurav-mtalkz) [inline src/pages/organization/update/[updateID].js:164]: Phone is optional and can contain between 7 and 13 digits

File: src/pages/organization/update/[updateID].js [hunk 7]
```diff
@@ -163,8 +189,50 @@ const updateID = () => {
...
+    )}
+   </div>
+
+   <div>
+   <span className="text-default">Reseller</span>
+   <Switch
+     onChange={() => handleChange(!checked)}
+     checked={checked}
+     handleDiameter={24}
+     uncheckedIcon={false}
```
Comment (gaurav-mtalkz) [inline src/pages/organization/update/[updateID].js:228]: show checked and unchecked icons. do it for all switches, unless they have special labels

File: src/pages/role/addRole.js [hunk 8]
```diff
@@ -150,6 +172,12 @@ const [permissions,setPermissions]=useState([])
        className="react-switch"
```
Comment (gaurav-mtalkz) [inline src/pages/role/addRole.js:172]: show checked and unchecked icons

File: src/pages/role/addRole.js [hunk 9]
```diff
@@ -150,6 +172,12 @@ const [permissions,setPermissions]=useState([])
...
        </label>
+       {errors && errors.is_system_role && (
+        errors.is_system_role.map((err)=>{
+         return <p className="mt-1 text-xs text-red-500">{err}</p>
+        })
+       
+      )}
      </div>
 
      <div style={{ width: "300px" }}>
```
[the remark this answers is not in the document]
Comment (keerthi-mtalkz) [reply src/pages/role/addRole.js:183]: if is system role is on  we need to hide the organization selector and if off we have to show the selector???

**Review by gaurav-mtalkz — APPROVED** [review]

**Pull request merged**